source/common/ur_util.h

```diff
@@ -76,4 +80,54 @@ inline bool getenv_tobool( const char* name )
     return 0 == strcmp( "1", env );
 }
 
+inline int parse_env_var(const char *env_var_name,
```

Comment:
I'm not sure about the name, it doesn't say to what the env is parsed, perhaps `getenv_to_map` would be better?